apps/preview/next/pages/showcases/Combobox/ComboboxBasic.tsx

```diff
@@ -24,11 +24,11 @@ export function DisableSwitch({
   className: string;
 }) {
   return (
-    <div className={classNames(['mt-40', className])}>
+    <div className={classNames(['mt-4', className])}>
       {/* eslint-disable-next-line jsx-a11y/label-has-associated-control */}
       <label className="flex items-center">
         <SfSwitch checked={disabledState} value="disabled" onChange={() => setDisabledState(!disabledState)} />
```

Comment:
If you want to set next state based on the previous state, you should use option to pass a callback to the state setter function. https://react.dev/reference/react/useState#setstate
```suggestion
        <SfSwitch checked={disabledState} value="disabled" onChange={() => setDisabledState((disabled) => !disabled)} />
```